archlinux/dist

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -o errexit -o pipefail
+
+export cpuCount=$(awk '/^processor/{n+=1}END{print n}' /proc/cpuinfo)
```

Comment:
```suggestion
cpuCount=`grep -c -w ^processor /proc/cpuinfo`
```

Don't `export` unless this env var is needed in sub processes. And avoid `$()` by using backticks unless you need a fully featured subshell inside.

Reply:
I did not now the part of why not to use $() thanks for that. I will remove the export then and change it use backticks.

Reply:
Changed it.